bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/i18n/LocaleProvider.java

```diff
@@ -25,4 +28,10 @@
      */
     Locale getLocale();
 
+    /**
+     * Provides access to the location of the installation
+     *
+     * @return location of the current installation or null if the location is not set
+     */
+    PointType getLocation();
```

Comment:
Maybe I miss some context but it seems unexpected that a `LocaleProvider` in package `i18n` does something with a location instead of strictly with locals maybe some bigger refactoring is needed to make this actually a `ConfigurationProvider` in a more generic package.